src/motion/__tests__/variant.test.tsx

```diff
@@ -39,474 +39,474 @@ describe("animate prop as variant", () => {
-        const onUpdate = jest.fn()
-
-        await new Promise(resolve => {
-            const x = motionValue(0)
-            const Component = ({ xTarget = 0 }) => (
-                <motion.div
-                    animate={{ x: xTarget }}
-                    transition={{ type: false }}
-                    onUpdate={onUpdate}
-                    style={{ x }}
-                />
-            )
-
-            const { rerender } = render(<Component xTarget={0} />)
-            setTimeout(() => rerender(<Component xTarget={1} />), 30)
-            setTimeout(() => rerender(<Component xTarget={1} />), 60)
-            setTimeout(() => resolve(), 90)
-        })
-
-        expect(onUpdate).toHaveBeenCalledTimes(1)
-    })
-
-    test("accepts variants without being typed", () => {
-        expect(() => {
-            const variants = {
-                withoutTransition: { opacity: 0 },
-                withJustDefaultTransitionType: {
-                    opacity: 0,
-                    transition: {
-                        duration: 1,
-                    },
-                },
-                withTransitionIndividual: {
-                    transition: {
-                        when: "beforeChildren",
-                        opacity: { type: "spring" },
-                    },
-                },
-                withTransitionType: {
-                    transition: {
-                        type: "spring",
-                    },
-                },
-                asResolver: () => ({
-                    opacity: 0,
-                    transition: {
-                        type: "physics",
-                        delay: 10,
-                    },
-                }),
-                withTransitionEnd: {
-                    transitionEnd: { opacity: 0 },
-                },
-            }
-            render(<motion.div variants={variants} />)
-        }).not.toThrowError()
-    })
-
-    test("new child items animate from initial to animate", () => {
-        const x = motionValue(0)
-        const Component = ({ length }: { length: number }) => {
-            const items = []
-            for (let i = 0; i < length; i++) {
-                items.push(
-                    <motion.div
-                        key={i}
-                        variants={variants}
-                        style={{ x: i === 1 ? x : 0 }}
-                    />
-                )
-            }
-
-            return (
-                <motion.div initial="hidden" animate="visible">
-                    <motion.div>{items}</motion.div>
-                </motion.div>
-            )
-        }
-
-        const { rerender } = render(<Component length={1} />)
-        rerender(<Component length={1} />)
-        rerender(<Component length={2} />)
-        rerender(<Component length={2} />)
-
-        expect(x.get()).toBe(100)
-    })
+    // test("fires onAnimationStart when animation begins", async () => {
```

Comment:
😂 What's the deal with these tests? Not valuable anymore? Or just not compatible with Visual Element yet and need to be updated down the road?

Reply:
Ha no I forgot to uncomment them. I was testing just the one test at the time.